backend/core/src/listings/listings.service.ts

```diff
@@ -79,6 +79,7 @@ export class ListingsService {
       // and substitues for the `:paramName` placeholders in the WHERE clause.)
       .setParameters(innerFilteredQuery.getParameters())
       .orderBy(getOrderByCondition())
+      .addOrderBy("summaryUnitType.num_bedrooms", "ASC", "NULLS LAST")
```

Comment:
Will this still be broken for upstream, if they don't use the unit summary table?

Reply:
No, they won't have been affected: they sort when they create `unitsSummarized` (example: https://github.com/bloom-housing/bloom/blob/93a06f8f886b3d0b6214a037d2067d5675f357cc/backend/core/src/shared/units-transformations.ts#L318-L323).